proto/info/info.proto

```diff
@@ -1348,4 +1348,7 @@ message ZInfoKubeCluster {
 
   // Info message on cluster storage
   KubeStorageInfo storage = 4;
+
+  // Cluster uuid
+  string cluster_id = 5;
```

Comment:
Is this the UUID assigned by the controller for the edge node cluster object, or is it some uuid generated by k3s? It makes sense to clarify in the comment what it is.

Reply:
this should be the same id passed to the edge-node in the config message EdgeNodeCluster, I'll update the comment.